diff -Nur o.fwbuilder-2.1.12/src/src.pro n.fwbuilder-2.1.12/src/src.pro
--- o.fwbuilder-2.1.12/src/src.pro	2007-06-07 04:33:54.000000000 +0200
+++ n.fwbuilder-2.1.12/src/src.pro	2007-06-28 01:10:37.000000000 +0200
@@ -10,6 +10,8 @@
 TARGET   = src
 SUBDIRS  = res fwbedit fwblookup
 # NOTE: 2.1 does not install scripts from src/tools ( fwb_install , fwb_compile_all)
+# but PLD do this 
+SUBDIRS	+= tools
 
 contains( HAVE_ANTLR_RUNTIME, 1 ) {
   SUBDIRS += antlr parsers
--- fwbuilder-3.0.5-orig/qmake.inc.in	2009-06-24 18:59:55.000000000 +0200
+++ fwbuilder-3.0.5/qmake.inc.in	2009-07-15 14:34:56.350185928 +0200
@@ -58,7 +58,7 @@ unix {
 
         PKGLOCALEDIR = $$res.path/locale
 
-        CONFIG += warn_on debug
+        CONFIG += warn_on
         QMAKE_CFLAGS_DEBUG += -Wno-unused-parameter
         QMAKE_CFLAGS_RELEASE += -Wno-unused-parameter
         QMAKE_CXXFLAGS_DEBUG += -Wno-unused-parameter @CXXFLAGS@
--- fwbuilder-3.0.5-orig/doc/doc.pro	2009-06-24 18:59:48.000000000 +0200
+++ fwbuilder-3.0.5/doc/doc.pro	2009-07-15 14:36:50.350151695 +0200
@@ -48,9 +48,8 @@ man.files   = fwbedit.1             \
               fwb_ipt.1             \
               fwb_pf.1              \
               fwb_pix.1             \
-
-#             fwb_install.1           \
-#			  fwb_compile_all.1       \
+              fwb_install.1           \
+              fwb_compile_all.1       \
 
 man.path    = $$MANDIR/man1
 
