--- fwbuilder-4.1.3/src/libgui/utils.h.orig	2010-12-06 02:29:02.000000000 +0200
+++ fwbuilder-4.1.3/src/libgui/utils.h	2010-12-09 16:36:32.000000000 +0200
@@ -47,7 +47,7 @@
 #include "fwbuilder/FWObject.h"
 #include "fwbuilder/FWReference.h"
 
-typedef std::pair<const QString,QString> QStringPair;
+typedef std::pair<QString,QString> QStringPair;
 
 // a predicate used to compare first string in pair<QString,QString>
 // use with find_if
@@ -55,7 +55,7 @@
     QString str;
     public:
     findFirstInQStringPair(const QString &d) { str=d; }
-    bool operator()(std::pair<const QString,QString> &_d)
+    bool operator()(std::pair<QString,QString> &_d)
     { return (str == _d.first); }
 };
 
@@ -63,7 +63,7 @@
     QString str;
     public:
     findSecondInQStringPair(const QString &d) { str=d; }
-    bool operator()(std::pair<const QString,QString> &_d)
+    bool operator()(std::pair<QString,QString> &_d)
     { return (str == _d.second); }
 };
 
